### PR TITLE
include print image in the Image API

### PR DESCRIPTION
**Pull request by edgarriba**
#### Changes

image

#### Type of change

- [ ] 📚  Documentation Update
- [ ] 🧪 Tests Cases
- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔬 New feature (non-breaking change which adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 This change requires a documentation update

#### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Did you update CHANGELOG in case of a major change?

### CODE REVIEW

File: kornia/image/image.py [hunk 1]
```diff
@@ -267,3 +278,18 @@ def write(self, file_path: str | Path) -> None:
         if self.channels_order == ChannelsOrder.CHANNELS_LAST:
             data = data.permute(2, 0, 1)
         write_image(file_path, data)
+
+    def print(self, max_width: int = 256) -> None:
```
Comment (shijianjian) [inline kornia/image/image.py:282]: Probably add `os.get_terminal_size().columns` as the default?

Comment (edgarriba) [reply kornia/image/image.py:282]: mmm, tried and not working very well

**Review by johnnv1 — APPROVED** [review]

**Comment by johnnv1** [comment]
the build failling is related to this PR
```console
__________________ [doctest] kornia.image.image.Image.print ___________________
284 
285         .. image:: https://github.com/kornia/data/blob/a82d2bdc868f3774a1be91419a2870e54222389e/print_image.png
286 
287         Args:
288             max_width: the maximum width of the image to print.
289 
290         Example:
291             >>> data = np.ones((4, 5, 3), dtype=np.uint8)  # HxWxC
292             >>> img = Image.from_numpy(data)
293             >>> img.print()
UNEXPECTED EXCEPTION: TypeError("Not a color or gray tensor. Got: <class 'torch.Tensor'>.\nNone")
Traceback (most recent call last):
  File "/usr/share/miniconda/envs/test/lib/python3.8/doctest.py", line 1336, in __run
    exec(compile(example.source, filename, "single",
  File "<doctest kornia.image.image.Image.print[2]>", line 1, in <module>
  File "/home/runner/work/kornia/kornia/kornia/image/image.py", line 295, in print
    print(image_to_string(self.data, max_width))
  File "/home/runner/work/kornia/kornia/kornia/utils/image_print.py", line 350, in image_to_string
    KORNIA_CHECK_IS_IMAGE(image, None, raises=True)
  File "/home/runner/work/kornia/kornia/kornia/core/check.py", line 362, in KORNIA_CHECK_IS_IMAGE
    res = KORNIA_CHECK_IS_COLOR_OR_GRAY(x, msg, raises=raises)
  File "/home/runner/work/kornia/kornia/kornia/core/check.py", line 338, in KORNIA_CHECK_IS_COLOR_OR_GRAY
    raise TypeError(f"Not a color or gray tensor. Got: {type(x)}.\n{msg}")
TypeError: Not a color or gray tensor. Got: <class 'torch.Tensor'>.
```

**Comment by edgarriba** [comment]
i'll fix that

**Pull request merged**